### PR TITLE
fix variants name for easier deploy

### PR DESCRIPTION
**Pull request by oto-ledger**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -24,10 +24,10 @@ include $(BOLOS_SDK)/Makefile.defines
 APP_LOAD_PARAMS=--appFlags 0x40 --path "2152157255'" --curve secp256k1 $(COMMON_LOAD_PARAMS) 
 
 ifeq ($(MULTISLOT),)
-MULTISLOT := 0
+MULTISLOT := openpgp_lite
```
Comment (cslashm) [inline Makefile]: It would had been better to keep XL convention
s/openpgp_lite/openpgp

File: Makefile [hunk 2]
```diff
@@ -24,10 +24,10 @@ include $(BOLOS_SDK)/Makefile.defines
...
 endif
 
-ifeq ($(MULTISLOT),0)
+ifeq ($(MULTISLOT),openpgp_lite)
```
Comment (cslashm) [inline Makefile]: It would had been better to keep XL convention
s/openpgp_lite/openpgp

File: Makefile [hunk 3]
```diff
@@ -67,7 +67,7 @@ all: default
 
 
 listvariants:
-	@echo VARIANTS MULTISLOT 0 1
+	@echo VARIANTS MULTISLOT openpgp_lite openpgp
```
Comment (cslashm) [inline Makefile]: s/openpgp_lite openpgp/openpgp openpgp_XL

**Comment by xchapron-ledger** [comment]
Already fixed

**Pull request closed without merging**